diff --git a/android/mesa3d_cross.mk b/android/mesa3d_cross.mk
index c3dec2e..634798f 100644
--- a/android/mesa3d_cross.mk
+++ b/android/mesa3d_cross.mk
@@ -179,8 +179,8 @@ endef
 
 define m-lld-flags-cleaned
   $(patsubst -Wl$(comma)--build-id=%,, \
-  $(subst prebuilts/,$(AOSP_ABSOLUTE_PATH)/prebuilts/, \
-  $(subst $(OUT_DIR)/,$(call relative-to-absolute,$(OUT_DIR))/, \
+  $(patsubst prebuilts/%,$(AOSP_ABSOLUTE_PATH)/prebuilts/%, \
+  $(patsubst $(OUT_DIR)/%,$(call relative-to-absolute,$(OUT_DIR))/%, \
   $(subst -Wl$(comma)--fatal-warnings,,                \
   $(subst -Wl$(comma)--no-undefined-version,,          \
   $(subst -Wl$(comma)--gc-sections,,                   \
